### PR TITLE
Bugfix/LIVE-2373 Fix zec parser for v5 transaction

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description

Fix zec parser for v5 transaction

### ❓ Context

- **Impacted projects**: `` None
- **Linked resource(s)**: `` https://ledgerhq.atlassian.net/browse/LIVE-2373

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 📸 Demo

No demo as it is a bug fix

### 🚀 Expectations to reach
User is able to spent v5 utxo transaction with Ledger Live

### CODE REVIEW

File: libs/ledgerjs/packages/hw-app-btc/src/splitTransaction.ts [hunk 1]
```diff
@@ -17,14 +17,16 @@ export function splitTransaction(
...
+  let witnessScript, locktime;
   const isDecred = additionals.includes("decred");
   const transaction = Buffer.from(transactionHex, "hex");
   const version = transaction.slice(offset, offset + 4);
   const overwinter =
     version.equals(Buffer.from([0x03, 0x00, 0x00, 0x80])) ||
-    version.equals(Buffer.from([0x04, 0x00, 0x00, 0x80]));
+    version.equals(Buffer.from([0x04, 0x00, 0x00, 0x80])) ||
+    version.equals(Buffer.from([0x05, 0x00, 0x00, 0x80]));
+  const zcashv5 = version.equals(Buffer.from([0x05, 0x00, 0x00, 0x80]));
```
Comment (haammar-ledger) [inline libs/ledgerjs/packages/hw-app-btc/src/splitTransaction.ts:28]: Maybe name it `isZcashV5` to be consistent with other variables.

File: libs/ledgerjs/packages/hw-app-btc/src/splitTransaction.ts [hunk 2]
```diff
@@ -100,18 +104,16 @@ export function splitTransaction(
...
     locktime = transaction.slice(transaction.length - 4);
-  } else {
+  } else if (!zcashv5) {
     locktime = transaction.slice(offset, offset + 4);
   }
 
   offset += 4;
 
-  if (overwinter || isDecred) {
+  if ((overwinter || isDecred) && !zcashv5) {
```
Comment (haammar-ledger) [inline libs/ledgerjs/packages/hw-app-btc/src/splitTransaction.ts:116]: Not very important but this condition looks weird, overwinter and zcash should probably be in the same condition.
Maybe: `if ((overwinter && !zcashv5) || isDecred)`
But really not important :smile:

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
@hzheng-ledger can you please add a changeset?
Also, do you think it's possible to cover this change with a test?
Thanks.

**Comment by hzheng-ledger** [comment]
Replace with https://github.com/LedgerHQ/ledger-live/pull/836

**Pull request closed without merging**